Userland/Libraries/LibWeb/CSS/StyleComputer.cpp

```diff
@@ -972,6 +1273,9 @@ ErrorOr<void> StyleComputer::compute_cascaded_values(StyleProperties& style, DOM
         did_match_any_pseudo_element_rules = true;
     }
 
+    // matching_rule_set.animation_rules = collect_matching_rules(element, CascadeOrigin::Animation, pseudo_element);
+    // sort_matching_rules(matching_rule_set.animation_rules);
```

Comment:
Is this meant to be a FIXME? Or did you just leave it here by mistake?

Reply:
Leftover from previous impl, will remove.